### PR TITLE
chromecast - Use new setVolumeByIncrement method from chromecast api …

### LINKED ISSUE #2026: [Chromecast] volume level not set correctly
When setting the volume ChromeCast does not allow you to set to a value larger than a specific percentage, thus requiring us to increment the value.   Result is you see the slider "bounce back" when setting the volume.  Also sending commands to set the volume from rules require you to manually increment the value.
https://developers.google.com/cast/docs/design_checklist/sender#sender-control-volume

I have created a pull request for the chromecast plugin here:  https://github.com/vitalidze/chromecast-java-api-v2/pull/72

I have also updated the chromecast plugin for openhab with the fix and will be submitting a pull request shortly.

### PR DESCRIPTION
**Pull request by abedwardsw**
chromecast - Use new setVolumeByIncrement method from chromecast api to avoid "bounceback" issue, closes #2026

Signed-off-by: Adam Edwards <abedwards@gmail.com> (github: abedwardsw)

### CODE REVIEW

**Comment by abedwardsw** [comment]
of note, the pull request on the upstream project was merged to master and approved. https://github.com/vitalidze/chromecast-java-api-v2/pull/72

File: addons/binding/org.openhab.binding.chromecast/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -7,7 +7,7 @@ Bundle-Version: 2.1.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ClassPath: .,
  lib/protobuf-java-2.6.0.jar,
- lib/api-v2-0.9.4-SNAPSHOT.jar
+ lib/api-v2-0.10.1-SNAPSHOT.jar
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.chromecast/META-INF/MANIFEST.MF]: I assume you have build this from your previous fork - as you did some changes after the review, would it make sense to create a new build from the official master?

Comment (abedwardsw) [inline addons/binding/org.openhab.binding.chromecast/META-INF/MANIFEST.MF]: changes were mostly cosmetic and didn't affect functionality.  Regardless I have rebuilt the jar locally from the latest master branch, I have rebased and pushed

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**